Moesif.Middleware/NetCore/MoesifMiddlewareNetCore.cs

```diff
@@ -23,12 +23,6 @@ public class MoesifMiddlewareNetCore
     {
         private readonly RequestDelegate _next;
 
-        public string userId;
-
-        public string companyId;
-
-        public string sessionToken;
-
         public Dictionary<string, object> metadata;
```

Comment:
Doesn't metadata also need to be moved?

Reply:
yeah..talked offline..he fixed it next commit

Reply:
ok. Thanks.